### DISCUSSION TITLE
A "heating-newbie" has some basic question fi you don't mind

### DISCUSSION THREAD

**Discussion by ovizii** [emsesp/EMS-ESP32#2136] [Q&A]
Hi everyone,

I have some pretty basic questions only partially related to EMS-ESP32, but I saw some pretty knowledgeable people on this discussion board, so I thought I'd try my luck. If there is a better place, like a specific Reddit or discord, please point me there too.

I'm renovating an old house and want central heating implemented using a gas boiler which also serves hot water. I need the system to have physical thermostats and able to work independently of my home assistant instance but also have the option of being controlled by HA. 

Since the house is in Eastern Europe, I am sometimes limited by what hardware is available, sometimes there are special models for this region meaning I struggle to find manuals and details.

**Current situation:**
- I would like not to have any cloud or mobile app dependency 
- 5 rooms, each should have its own wired wall thermostat (with zigbee)
- independent piping from the boiler room to each radiator in a separate circuit meaning I can send heat to each radiator separately
- no TRVs on the radiators, the water flow is managed centrally with valves locate near the boiler. Planning to use these valves T30NC-230V => https://saluscontrols.com/de/produkt/t30nc230-wp-thermischer-stellantrieb-230v/
- the person executing the works, suggested something called (I think) zone controller possibly a smaller version of the Salus KL08NSB => https://saluscontrols.com/de/produkt/kl08nsb-8-zonen-regelklemmleiste/ 

**So basically, the thermostats trigger the KL08NSB which then opens the T30NC-230V and triggers the boiler via the on/off wiring**

The idea was, that if I rent out this place, the heating can be controlled manually via a simple wall thermostat and if I am staying there, I can control the heating via HA.

**This is when I noticed the suggested boiler Bosch GC2200 W does have an EMS Bus and googling I found EMS-ESP32 and the option of connecting a gateway, so HA also gains access to the bus.**

**And here is the gap in my knowledge I'd love to fill:**
- I think that with my above described solution, manual controls of heating work fine and HA can also control it, but I think I might miss out on any benefits an EMS bus offer, is that right?
- HA can read/write to the boiler's setup using an EMS gateway but do I also need a real physical EMS capable wall thermostat or can all this be simulated? 
- what exactly is the purpose of having one of these real EMS Bus capable thermostats? They are mounted in a single room, I really don't get their role in all of this. I assume if I need to replace one of my wired+zigbee thermostats with a specific EMS Bus model, that one needs to be wired to the described solution with an on/off wire AND to the EMS Bus, not sure if that is really possible
- I thought it would be awesome if HA could gather the knowledge the boiler lacks i.e. how many thermostats are asking for heat, what is the desired/target temperature and how big is the difference between the target and the current temperature and feed this information to the boiler.

I am open to all tips and tricks and links to read more, I am really new to this subject of heating especially with EMS Bus.

P.S. The boiler seems to have an on/off AND EMS Bus connection according to the manual.
![image](https://github.com/user-attachments/assets/73ea3358-13b9-4fc8-b36d-915fc586b615)

**Comment by proddy** [comment]
@bbqkees any ideas on this one?

**Reply by ovizii** [reply]
Thank you so much for having a look. I am still reading anything related I can find but getting some answers to these questions would make my life much easier while navigating this new world of smart heating. :-)
